app/src/main/java/a3/com/convo/fragments/GameFragment.java

```diff
@@ -104,8 +108,23 @@ public void done(ParseUser object, ParseException e) {
                         allLikes.addAll(player2Likes);
                         Collections.shuffle(allLikes);
 
-                        adapter = new CardAdapter(allLikes, context);
+                        adapter = new CardAdapter(allLikes, GameFragment.this);
                         cardStack.setAdapter(adapter);
+
+                        // when a card is swiped, add it to topics discussed
+                        cardStack.setCallback(new SwipeDeck.SwipeDeckCallback() {
+                            @Override
+                            public void cardSwipedLeft(long stableId) {
+                                // reset the timer of the next card
+                                topicsDiscussed.add(allLikes.get((int)stableId));
```

Comment:
Do we need to cast it? If so we should make sure this does not cause any crashes

Reply:
I think we do because get() only takes an int, but I added a check